src/cloudant/design_document.py

```diff
@@ -522,6 +595,17 @@ def get_list_function(self, list_name):
         """
         return self.lists.get(list_name)
 
+    def get_show_function(self, show_name):
+        """
+        Retrieves a specific show function from the locally cached DesignDocument
+        shows dictionary by name.
+
+        :param str show_name: Name used to identify the show function.
+
+        :returns: Index dictionary for the specified show function name
```

Comment:
Updated in https://github.com/cloudant/python-cloudant/pull/196/commits/22c15e4a2de6c80755c6e1aa61fb6c44f95b9c83#diff-f2994c8b7af6529726f4b4fbdd48e119R594 and https://github.com/cloudant/python-cloudant/pull/196/commits/22c15e4a2de6c80755c6e1aa61fb6c44f95b9c83#diff-f2994c8b7af6529726f4b4fbdd48e119R594